base/applications/taskmgr/graphctl.c

```diff
@@ -407,6 +420,51 @@ void GraphCtrl_Paint(TGraphCtrl* this, HWND hWnd, HDC dc)
     DeleteDC(memDC);
 }
 
+void GraphCtrl_ShiftGrid(TGraphCtrl* this)
+{
+    RECT rectCleanUp;
+    HPEN oldPen;
+    HPEN solidPen = CreatePen(PS_SOLID, 0, this->m_crGridColor);
```

Comment:
As you're being given a newly returned handle to a pen, that handle will stay valid even after the completion of this function, ultimately leaking it. You must free that handle when you no longer need it.

Reply:
Added `DeleteObject(solidPen);`.